recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+cuda_compiler_version:                          # [linux64]
+  - None                                        # [linux64]
+docker_image:                                   # [linux64]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64  # [linux64]
```

Comment:
so there is no way to stay on `cos6`?

Reply:
No, we cannot, see [this](https://github.com/conda-forge/ray-packages-feedstock/pull/28#issuecomment-894279918) comment.

Reply:
But we can use older `nodejs`, e.g. one from `14.x` version part, which is still in "active" phase according to their schedule and works on `cos6` :)

Reply:
Ah, this comment didn't show up in the main thread for some reason.

I'm strongly -1 against pinning to old versions for the sake of EOL distros.

Reply:
If you feel so strongly about cos6-support, we can add a branch where nodejs is pinned to 14.x, but master must move on. (though this would need some care with the build numbers; e.g. build numbers on master start from 100)